contributors/devel/sig-testing/writing-good-conformance-tests.md

```diff
@@ -0,0 +1,56 @@
+# Writing Good Conformance Tests for Kubernetes #
+
+Conformance tests are promoted from existing e2e tests which have
+been proven to be stable and "non-flaky".
+
+Please read the the 
+["Writing good e2e
+tests"](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/writing-good-e2e-tests.md)
+guide as a supplment to this document.
+
+## Patterns and Anti-Patterns ##
+
+### Goals of Conformance tests ###
+
+The overarching goal of conformance tests is to exercise core
+Kubernetes functionality in (as much as possible) a provider-agnostic
+way. Conformance tests should not target internal behaviors, but
+instead target exposed system behaviors. 
+
+#### Requirements  ####
+
+In addition to the "Writign good e2e tests" guide, there are specific
+requirements that all Conformance tests must adhere to. You can find
+the definitive list of requirements in the [SIG-Arch community
+docs](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#conformance-test-requirements).
+
+
+#### Existing Tests ####
+
+When looking for an existing e2e test which exercises a behaviour
+which is useful for Conformance, you may find that the test does not
+meet all of the [requirements](#Requirements). In these cases, you
+might either modify the test to meet requirements, but sometimes doing
+so will mean losing some of the intent of the test.
+
+Instead, a better approach may be to duplicate the test, and make your
+modifications, thus preserving the original test as-is.
+
+Any existing e2e test should  have gone through a standard review
+process by a respective [SIG](https://github.com/kubernetes-sigs) in order to
```

Comment:
I think it's better to link to the [sig-list](https://github.com/kubernetes/community/blob/master/sig-list.md) file under community, since it's a bit easier to find things in this nice, alphabetized list.